rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/MethodDeclarationTest.java

```diff
@@ -67,37 +67,37 @@ def accept(Map m) {
     }
 
     @Test
-    void methodDeclarationWithModifiers() {
+    void primitiveReturn() {
```

Comment:
All the test changes look a little ugly, but now all the kind of same tests are grouped together.